open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/src/main/java/org/odpi/openmetadata/userinterface/uichassis/springboot/auth/SecurityConfig.java

```diff
@@ -25,14 +25,19 @@ protected void configure(HttpSecurity http) throws Exception {
             .exceptionHandling().and()
             .anonymous().and()
             .authorizeRequests()
-            .antMatchers("/**").permitAll()
+            .antMatchers("/").permitAll()
             .antMatchers("/error/**").permitAll()
+            .antMatchers("/node_modules/**").permitAll()
+            .antMatchers("/src/**").permitAll()
             .antMatchers("/h2/**").permitAll()
-            .antMatchers("/api/**").authenticated()
             .antMatchers("/api/css/**").permitAll()
             .antMatchers("/api/js/**").permitAll()
-            .antMatchers("/api/auth/login").permitAll()
```

Comment:
This shouldn't be removed
It is the authentication request

Reply:
this file should not be changed

/** -> makes sure it gets everything that can be in the UI(in a different server, you can't just add them all here)
/api -> makes sure that it gets everything that can be in the API

the `/api/css/**`and `/api/js/**`are there because it's a config thing that comes from the API

please revert this file

Reply:
reverting this file will cause all the requests to pass through without any authentifcation.

Reply:
`.antMatchers("/**").permitAll()` -> allows all

`.antMatchers("/api/**").authenticated()` -> filters all so that the `/api/**` requests are authenticated